src/plugins/irc-events/error.js

```diff
@@ -10,6 +10,16 @@ module.exports = function(irc, network) {
 
 		if (data.reason) {
 			text = data.reason + " (" + text + ")";
+			if (data.channel) {
+				text = data.channel + ": " + text;
+			}
+		}
+
+		if (data.error === "user_on_channel") {
+			text = data.channel + ": User is already on channel (" + data.error + ")";
+			if (data.nick) {
+				text = data.channel + ": " + data.nick + " is already on channel (" + data.error + ")";
```

Comment:
Would be nicer to combine this into same message above, for example `User (${data.nick}) is...`.

You can use `+=` to cut text duplication.